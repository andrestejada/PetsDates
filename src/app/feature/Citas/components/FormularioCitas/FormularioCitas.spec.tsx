import '@testing-library/jest-dom';
import {  ShallowWrapper,shallow } from 'enzyme';
import { FormularioCitas } from './index'; 
import React from 'react';




describe('Prueba unitarias formulario de citas', () => {
    let componentWrapper : ShallowWrapper;
   
    it('Comprobar que el componente se renderize correctamente ', () => {
        componentWrapper = shallow(<FormularioCitas/>);
        expect(componentWrapper).toMatchSnapshot();
    });
});

// describe('Prueba unitarias formulario de citas',()=>{
//     test('Comprobar que el componente se renderize correctamente',()=>{
//         // const wrapper = shallow(<FormularioCitas/>);
//         // expect(wrapper).toMatchSnapshot();
//     });
// });

