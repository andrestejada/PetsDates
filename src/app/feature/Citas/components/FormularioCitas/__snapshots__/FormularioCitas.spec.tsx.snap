// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba unitarias formulario de citas Comprobar que el componente se renderize correctamente  1`] = `
<form
  className="sc-AxjAm iHUMxp"
  onSubmit={[Function]}
>
  <div
    className="MuiFormControl-root MuiTextField-root"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
      data-shrink={false}
      htmlFor="outlined-required"
      id="outlined-required-label"
    >
      Nombre del Propietario
    </label>
    <div
      className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        autoFocus={false}
        className="MuiInputBase-input MuiOutlinedInput-input"
        disabled={false}
        id="outlined-required"
        name="nombrePropietario"
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        type="text"
        value=""
      />
      <fieldset
        aria-hidden={true}
        className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
      >
        <legend
          className="PrivateNotchedOutline-legendLabelled-3"
        >
          <span>
            Nombre del Propietario
          </span>
        </legend>
      </fieldset>
    </div>
  </div>
  <div
    className="MuiFormControl-root MuiTextField-root"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
      data-shrink={false}
      htmlFor="outlined-required"
      id="outlined-required-label"
    >
      Nombre de la mascota
    </label>
    <div
      className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        autoFocus={false}
        className="MuiInputBase-input MuiOutlinedInput-input"
        disabled={false}
        id="outlined-required"
        name="nombreMascota"
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        type="text"
        value=""
      />
      <fieldset
        aria-hidden={true}
        className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
      >
        <legend
          className="PrivateNotchedOutline-legendLabelled-3"
        >
          <span>
            Nombre de la mascota
          </span>
        </legend>
      </fieldset>
    </div>
  </div>
  <div
    className="MuiFormControl-root"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
      data-shrink={false}
      htmlFor="outlined-age-native-simple"
    >
      Tipo de servicio
    </label>
    <div
      className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
      onClick={[Function]}
    >
      Array [
        <select
          aria-invalid={false}
          className="MuiSelect-root MuiSelect-select MuiSelect-outlined MuiInputBase-input MuiOutlinedInput-input"
          disabled={false}
          multiple={false}
          name="tipoServicio"
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          value=""
        >
          <option
            aria-label="None"
            value=""
          />
          <option
            value="basico"
          >
            Basico
          </option>
          <option
            value="regular"
          >
            Regular
          </option>
          <option
            value="premium"
          >
            Premium
          </option>
        </select>,
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiSelect-icon MuiSelect-iconOutlined"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M7 10l5 5 5-5z"
          />
        </svg>,
      ]
      <fieldset
        aria-hidden={true}
        className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
        style={
          Object {
            "paddingLeft": 8,
          }
        }
      >
        <legend
          className="PrivateNotchedOutline-legend-2"
          style={
            Object {
              "width": 0.01,
            }
          }
        >
          <span
            dangerouslySetInnerHTML={
              Object {
                "__html": "&#8203;",
              }
            }
          />
        </legend>
      </fieldset>
    </div>
  </div>
  <div
    className="MuiFormControl-root MuiTextField-root"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-outlined MuiFormLabel-filled"
      data-shrink={true}
      htmlFor="outlined-required"
      id="outlined-required-label"
    >
      Tarifa
    </label>
    <div
      className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        autoFocus={false}
        className="MuiInputBase-input MuiOutlinedInput-input"
        disabled={false}
        id="outlined-required"
        name="tarfia"
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        readOnly={true}
        required={false}
        type="text"
        value={0}
      />
      <fieldset
        aria-hidden={true}
        className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
      >
        <legend
          className="PrivateNotchedOutline-legendLabelled-3 PrivateNotchedOutline-legendNotched-4"
        >
          <span>
            Tarifa
          </span>
        </legend>
      </fieldset>
    </div>
  </div>
  <div
    className="MuiFormControl-root MuiTextField-root"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink"
      data-shrink={true}
      htmlFor="datetime-local"
      id="datetime-local-label"
    >
      Fecha y hora de la cita
    </label>
    <div
      className="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
      onClick={[Function]}
    >
      <input
        aria-invalid={false}
        autoFocus={false}
        className="MuiInputBase-input MuiInput-input"
        disabled={false}
        id="datetime-local"
        name="fechaHora"
        onAnimationStart={[Function]}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        required={false}
        type="datetime-local"
        value=""
      />
    </div>
  </div>
  <div
    className="MuiFormControl-root MuiTextField-root"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-outlined"
      data-shrink={false}
      htmlFor="outlined-multiline-static"
      id="outlined-multiline-static-label"
    >
      Observaciones
    </label>
    <div
      className="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-multiline MuiOutlinedInput-multiline"
      onClick={[Function]}
    >
      Array [
        <textarea
          aria-invalid={false}
          autoFocus={false}
          className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMultiline MuiOutlinedInput-inputMultiline"
          disabled={false}
          id="outlined-multiline-static"
          name="observaciones"
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          rows={1}
          style={
            Object {
              "height": 0,
              "overflow": "hidden",
            }
          }
          value=""
        />,
        <textarea
          aria-hidden={true}
          className="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputMultiline MuiOutlinedInput-inputMultiline"
          readOnly={true}
          style={
            Object {
              "height": 0,
              "left": 0,
              "overflow": "hidden",
              "position": "absolute",
              "top": 0,
              "transform": "translateZ(0)",
              "visibility": "hidden",
            }
          }
          tabIndex={-1}
        />,
      ]
      <fieldset
        aria-hidden={true}
        className="PrivateNotchedOutline-root-1 MuiOutlinedInput-notchedOutline"
      >
        <legend
          className="PrivateNotchedOutline-legendLabelled-3"
        >
          <span>
            Observaciones
          </span>
        </legend>
      </fieldset>
    </div>
  </div>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
    data-testid="submit"
    disabled={false}
    onBlur={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="submit"
  >
    <span
      className="MuiButton-label"
    >
      Guardar Citas
    </span>
    <span
      className="MuiTouchRipple-root"
    />
  </button>
</form>
`;
